### DISCUSSION TITLE
Elevation Problem when using RustDesk with Service disabled

### DISCUSSION THREAD

**Discussion by Sven1403** [rustdesk/rustdesk#5861] [Feature Request]
### Bug Description

Hello,

we installed RustDesk in our Company yesterday. We first used portable but want to install it now since we had this problem: https://www.reddit.com/r/rustdesk/comments/142pn3j/pre_allowing_in_windows_firewall/

The Problem: We need to disable the RustDesk Service with Group Policy because we dont want to accidentally ask for a access on a computer in our enviroment. Connecting works fine but now we got Elevation Problems. With Service enabled you cant ask for elevation and you can just type your admin credentials in any UAC Window. But with Service disabled we got a problem like this: https://github.com/rustdesk/rustdesk/issues/5271
So you cant type in any UAC Windows and when ask for elevation you get this all the time:
![grafik](https://github.com/rustdesk/rustdesk/assets/77511450/700eb23a-5466-49e2-be79-59dd23daebc1)

You want RustDesk behave like this? No way to get Elevation with the Service disabled?

Thanks for any Informations! :)

### How to Reproduce

Disable RustDesk Service -> you are able to ask for Elevation but get always this error message
![grafik](https://github.com/rustdesk/rustdesk/assets/77511450/2341d451-c96d-4c53-871e-25960b3e300e)

Enable RustDesk Service -> You cant ask for Elevation but can type in UAC windows (so you have Elevation directly)

### Expected Behavior

Have Elevation directly with RustDesk Service disabled or are able to request it without the error message "no need to elevate"

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.2 -> 1.2.2

### Screenshots

![grafik](https://github.com/rustdesk/rustdesk/assets/77511450/c65ce64f-3ef6-4254-97af-f1741458605f)

**Comment by rustdesk** [comment]
Please do not challenge the way using RustDesk.

**Reply by rustdesk** [reply]
Mark this as feature request, we won't take action before receiving enough feedback from the other users.

**Reply by Sven1403** [reply]
Okay. Thank you!

**Reply by Sovince37** [reply]
@rustdesk :
Same problem with the version 1.2.4 (nightly) : "no need to elevate" with the service disabled by default, and the users who dont have admin rights.
Is it possible to :
- display UAC when the service is disabled (with rustdesk installed and running) or
- ask for the elevation rights (without the error message "no need to elevate")

**Comment by DoTheEvo** [comment]
Well I think I face this issue.

I want rustdesk disabled by default, not calling home every few seconds and being kinda exposed, DISABLED.

So I thought that to disabling service is for that, and it is.

But once user calls and we tell them to run rustdesk from desktop we dont have admin credentials even if the user got them.
we cant elevate, rustdesk is running but its running in some different mode and we cant get elevation unless we nagivate user to go to services, rustdesk, and start the service.

I feel rustdesk should check if its installed on the system and start the service and show gui, not just start in non-installed, no admin privilages mode.

**Reply by Sven1403** [reply]
Yeah you got it. I think there should be a option for using the installed version of RustDesk without the service running and then you should be able to request for elevation like in the portable version. I mean why is the button appearing in the INSTALLED version when you disable the service? If the button is there, i guess the elevation function should be usable or? @rustdesk
